src/schema.ts

```diff
@@ -233,6 +233,17 @@ const UserSchema = new Schema({
     enum: ["active", "locked"],
     default: "active",
   },
+
+  twoFactor: {
+    secret: {
+      type: String,
+      default: undefined,
```

Comment:
Why a default is needed here?

Reply:
Good question. I just removed it and it doesn't seem to cause any issues. I don't remember the exact error I solved by adding this, but it seems it's not there anymore.